contracts/Remittance.sol

```diff
@@ -87,11 +86,11 @@ contract Remittance is Stoppable {
     }
     
     function withdrawFees() public _onlyOwner returns(bool success) {
```

Comment:
`_onlyOwner`, so if I am a former owner, I cannot withdraw?

Reply:
I thought the point of this mapping was that if I was a former owner, I could withdraw my fees after the ownership transfer without having to time my withdraw and ownerChange transactions so that they are close to each other.

Reply:
I forgot to delete the modifier when replacing the uint by a mapping